core/contracts/financial_templates/PricelessPositionManager.sol

```diff
@@ -11,149 +11,173 @@ import "../Finder.sol";
 import "../OracleInterface.sol";
 /**
  * @title Financial contract with priceless position management.
- * @dev Handles positions for multiple sponsors in an optimistic (i.e., priceless) way without relying on a price feed.
+ * @author umaproject.org
```

Comment:
We haven't previously used these author lines. Just wondering what others think of starting to use them everywhere.

Reply:
ok, cool. thanks for the heads up. I removed it for consistency. can add it to all contracts later if we want.